source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -1851,7 +1854,7 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\software\policies\microsoft\office\16.0\outlook\options\mail
 
-If the value junkmailprotection is set to "High", this is not a finding.</RawString>
+If the value junkmailprotection is set to "3", this is not a finding.</RawString>
```

Comment:
In the same way as the comment above, this rule should be evaluated as well, since it changed.  Admx compared to registry afterward.

Reply:
Set via gpedit

![proplusemail](https://user-images.githubusercontent.com/20539446/128454542-e9f7b638-3cb1-47a6-89e9-c1ea0d2d3d0d.JPG)

Reply:
corrected hard coded rule